### PR TITLE
Add Souliss types 54, 55, 56 and 58 (light, voltage, current, pressure).

### PR DESCRIPTION
**Pull request by slepp**
Simple addition of Souliss types that are missing in the analog input typicals.

### CODE REVIEW

**Comment by teichsta** [comment]
any comments @fazioa?

File: bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT54.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.souliss.internal.network.typicals;
+
+/**
+ * Typical T54 Lux Sensor Derived from T51 Analog input, half-precision
+ * floating point
+ * 
+ * @author Tonino Fazio
+ * @since 1.7.0o
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT54.java]: 1.8.0

File: bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT54.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.souliss.internal.network.typicals;
+
+/**
+ * Typical T54 Lux Sensor Derived from T51 Analog input, half-precision
+ * floating point
+ * 
+ * @author Tonino Fazio
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT54.java]: please add you as the author and remove Tonino

File: bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT55.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.souliss.internal.network.typicals;
+
+/**
+ * Typical T55 Voltage Sensor Derived from T51 Analog input, half-precision
+ * floating point
+ * 
+ * @author Tonino Fazio
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT55.java]: please add you as the author and remove Tonino

File: bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT55.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+ * @since 1.7.0o
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT55.java]: 1.8.0

File: bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT56.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.souliss.internal.network.typicals;
+
+/**
+ * Typical T56 Current Sensor Derived from T51 Analog input, half-precision
+ * floating point
+ * 
+ * @author Tonino Fazio
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT56.java]: please add you as the author and remove Tonino

File: bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT56.java [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+ * @since 1.7.0o
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT56.java]: 1.8.0

File: bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT58.java [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.souliss.internal.network.typicals;
+
+/**
+ * Typical T58 Pressure Sensor Derived from T51 Analog input, half-precision
+ * floating point
+ * 
+ * @author Tonino Fazio
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT58.java]: please add you as the author and remove Tonino

File: bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT58.java [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+ * @since 1.7.0o
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/network/typicals/SoulissT58.java]: 1.8.0

**Comment by teichsta** [comment]
Thanks @slepp. Please find some review comments inline …

**Comment by slepp** [comment]
@teichsta Updated per the comments inline.

**Pull request merged**